conf.d/main

```diff
@@ -6,12 +6,12 @@ DB_PASS=$(mcookie)
 
 SRC=/usr/local/src
 WEBROOT=/var/www/opencart
-STORAGE=$(dirname $WEBROOT)/opencart-storage
+STORAGE=/var/www/opencart-storage
 
 DOMAIN=www.example.com
 
 unzip $SRC/opencart-*.zip -d $SRC/opencart
-mv $SRC/opencart/upload $WEBROOT
+mv $(find $SRC -type d -name "opencart-*")/upload $WEBROOT
```

Comment:
IIRC `unzip` with `-d DIRNAME` will dump all files in the zip straight into DIRNAME - so this line should have worked too:
```
mv $SRC/opencart-*/upload $WEBROOT
```

Again, don't worry, just wanted to share...